lib/adiwg/mdtranslator/readers/iso19115_2/modules/module_security_constraint.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'nokogiri'
+require 'adiwg/mdtranslator/internal/internal_metadata_obj'
+
+module ADIWG
+   module Mdtranslator
+      module Readers
+         module Iso191152
+            module SecurityConstraint
+               @@securityConstraintXPath = 'gmd:MD_SecurityConstraints'
+               @@classificationXPath = 'gmd:classification//gmd:MD_ClassificationCode'
+               @@userNoteXPath = 'gmd:userNote//gco:CharacterString'
+               @@classSysXPath = 'gmd:classificationSystem//gco:CharacterString'
+               @@handlingXPath = 'gmd:handlingDescription//gco:CharacterString'
+               @@type = 'security'
+               def self.unpack(xSecurityConstraint, hResponseObj)
+                  intMetadataClass = InternalMetadata.new
+                  hConstraint = intMetadataClass.newConstraint
+                  hSecConstraint = intMetadataClass.newSecurityConstraint
+
+                  hConstraint[:type] = @@type
+
+                  # xSecurityConstraint = xConstraint.xpath(@@securityConstraintXPath)[0]
+                  return nil if xSecurityConstraint.nil?
```

Comment:
do we still want/need this check @rshewitt ?

Reply:
no we can remove it since we're doing that in `Constraint.unpack`